### ISSUE TITLE
Update assembly_higher_ranks.tsv

### ISSUE THREAD

**Issue by mdoering** [Sp2000/colplus-repo#17]
For the Staphbase superfamily `Hydrophiloidea` the parent is the class Insecta, not the expected order Coleoptera. That might be a bug valid for all superfamilies, it definitely happens more often.

Re-export from latest Assemlby_Global the `assembly_higher_ranks.tsv` file

**Comment by mdoering** [comment]
@gdower I'm seeing some of the old bad id issues again due to 2 capitals for the genus:

```
ERROR:  insert or update on table "_names" violates foreign key constraint "_names_parent_id_fkey"
DETAIL:  Key (parent_id)=(Animalia_Arthropoda_Insecta_Diptera_Simuliidae_SImulium) is not present in table "_names".
```

**Comment by gdower** [comment]
Fixed in 5448b26 and c17b3b43fbd801349412b016d1b5717e4bd8ee85
